src/frontend/src/components/modal.ts

```diff
@@ -0,0 +1,59 @@
+import { html, TemplateResult, render } from "lit-html";
+
+export const createModal = ({
+  slot,
+  id,
+  confirmText = "Ok",
+}: {
+  slot: TemplateResult;
+  id?: string;
+  confirmText?: string;
```

Comment:
I think we can leave those two out until we have a good use case for them

Reply:
do all the buttons say OK ?